pkg/apis/harvesterhci.io/v1beta1/backup.go

```diff
@@ -190,8 +143,12 @@ type VirtualMachineRestoreSpec struct {
 	// initially only VirtualMachine type supported
 	Target corev1.TypedLocalObjectReference `json:"target"`
 
+	// +kubebuilder:validation:Required
 	VirtualMachineBackupName string `json:"virtualMachineBackupName"`
 
+	// +kubebuilder:validation:Required
+	VirtualMachineBackupNamespace string `json:"virtualMachineBackupNamespace"`
```

Comment:
then the restore will be failed with a proper message like `can't find vmbackup balabala`, additionally, for now users can only restore VM with the same namespace of the backup, this is because PVC only support [VolumeSnapshot](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#volume-snapshot-and-restore-volume-from-snapshot-support) with the local source object.

in regards to https://github.com/harvester/harvester/issues/1027, this will be addressed separately as currently manually creating a `VolumeSnapshot` in a different namespace using the same `VolumeSnapshot` object has a bug on the LH side.